src/test/java/tests/AP_5_Test.java

```diff
@@ -19,5 +21,9 @@ void addProductsInCart() {
         Customers customers = new Customers();
         customers.setAddition(new AdditionInCart());
         customers.executeAddition();
+
+        InCartPage inCartPage = new InCartPage();
+        Assertions.assertEquals(inCartPage.totalPriceProductsText(),  inCartPage.sumProductPrice());
+        Assertions.assertEquals(inCartPage.counterProductText(),  "3 Products");
```

Comment:
just validate equals with 3, cause if they change naming... you will have an error.